src/java_bytecode/Makefile

```diff
@@ -39,15 +39,17 @@ include ../common
 
 CLEANFILES = java_bytecode$(LIBEXT) java_core_models.c java_core_models.h
 
-all: subsystem run java_bytecode$(LIBEXT)
+all: subsystem run core-models java_bytecode$(LIBEXT)
```

Comment:
Commit can be squashed into https://github.com/cesaro/cbmc/pull/5/commits/068c730d5164434a1b3c5894d57b04e361cef962